css/main.css

```diff
@@ -0,0 +1,406 @@
+  display: inline-block;
+  position: relative;
+  width: 100px;
+  height: 35px;
+  text-align: center;
+}
+
+
+.navigation-link:link,
+.navigation-link:visited{
+text-decoration: none;
+color: white;
+}
+
+.navigation-ul-list:not(:last-child) {
+  border-width: 0 2px 0 1px;
+  /* border-top-width: 0px; */
+  border-right-width: 2px;
+  /* border-bottom-width: 0px; */
+  border-left-width: 1px;
+  border-style: solid;
+  border-top-style: solid;
+  border-right-style: solid;
+  border-bottom-style: solid;
+  border-left-style: solid;
+  -o-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -ms-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -moz-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+  -webkit-border-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAABCAQAAABTNcdGAAAAEUlEQVQIHWP4r8bA8F%2BKwQEADZUCf37PGokAAAAASUVORK5CYII%3D) 0 2 0 1;
+}
+
+.navigation-ul-list:hover:not(:last-child) {
+  background: #575757;
+  background: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAAkCAQAAADifZiHAAAAJ0lEQ…zE8JeJ4R8q%2Bo%2BKCIH%2FjEA1TAyMSIgJRkIQM9BYAGa0Dj%2FXpUiAAAAAAElFTkSuQmCC) repeat-x 0 0, url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAABCAQAAACvIOyMAAAAE0lEQVQIW2P8z8jAwMAIh0wICAAa0gETv9XXOgAAAABJRU5ErkJggg%3D%3D) repeat-y 100% 0, url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAABCAQAAACvIOyMAAAAEklEQVQIW2P8L8TwDwn%2Bh0MGAKnjDAIoAAlcAAAAAElFTkSuQmCC) repeat-y 0 0, rgba(0, 0, 0, .4);
+}
+
+
+.navigation-ul-list:not(:first-child) {
+  margin-left: -4px;
+}
+
+
+.navigation-ul-list:first-child {
+  border-top-left-radius: 4px;
+  border-bottom-left-radius: 4px;
+}
+
+
+.navigation-span {
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%,-50%);
+}
+
+.navigation-icon {
+  color: white;
+  font-size: 1.4rem;
+}
+
+
+.navigation-search {
+   background: none;
+  position: relative;
+  top: -.8rem;
+  left: 1.5rem;
+  width: 100%;
+  padding: 2px 18px 1px 22px;
+  border-radius: 12px;
+  box-sizing: border-box;
+  background-color: grey rgba(255,255,255,.1);
+  /* -webkit-transition: color, text-shadow 0.3s, 0.3s;
+  transition: color, text-shadow 0.3s, 0.3s; */
+}
+
+.navigation-search:focus {
+  outline: none;
+  background-color: white;
+}
+
+
+.search-icon {
+  position: absolute;
+  top: -.5rem;
+  left: 2rem;
+  color: white;
+  z-index: 9000;
+
+}
+
+.navigation-search:focus + .search-icon {
+  color: grey;
+}
+
+
+
+/* styling the section-hero */
+
+.section-hero {
+  height: 90.4vh;
+  /* position: relative; */
+  /* background: #5e5e5e url(/img/hero_2x.jpg) 50% 100% no-repeat; */
+  /* background-size: 1300px 1315px; */
+  background-image: url('../img/hero_2x.jpg');
```

Comment:
Did not address `background` comment from before. Is it that you disagreed? That is fine if you did, but please state if you simply did not understand

Reply:
> Did not address `background` comment from before. Is it that you disagreed? That is fine if you did, but please state if you simply did not understand

Hi Maya, I don't know if I understood your comments on the issue of background in the last request. I am assuming you said "I could put the shorthand background property in one line and as such there was no need for the background-size property "? Since that was not working for the project, I had to use background-image property which gave me the feel I wanted for the project. but maybe you can explain further what you wanted me to do?

Reply:
Hi, yes this is fine! We talked about it on slack already 😄